README.adoc

```diff
@@ -103,7 +112,12 @@ A health report will be generated automatically for all services that enable Mic
 file.
 
 All services must provide an implementation of the `HealthCheck` interface, which will be used to
-verify their health.
+verify their health. MicroProfile Health offers health checks for both readiness and liveness. 
+A readiness check allows third-party services, such as Kubernetes, to know if the microservice 
+is ready to process requests. For example, checking dependencies such as a database connection. 
+A liveness check allows third-party services to determine if the microservice is running. 
+If the liveness check fails, this means that the application can be terminated. For example, 
+if the application runs out of memory. 
```

Comment:
Change
"A readiness check allows third-party services, such as Kubernetes, to know if the microservice is ready to process requests. For example, checking dependencies such as a database connection. A liveness check allows third-party services to determine if the microservice is running. If the liveness check fails, this means that the application can be terminated. For example, if the application runs out of memory. "
To
"A readiness check allows third-party services, such as Kubernetes, to determine whether a microservice 
is ready to process requests. For example, a readiness check might check dependencies, such as database connections. A liveness check allows third-party services to determine whether a microservice is running. If the liveness check fails, the application can be terminated. For example, a liveness check might fail if the application runs out of memory. "

**Definitely check this one for technical accuracy. Could be off base here.**